lib/astronoby/bodies/sun.rb

```diff
@@ -16,7 +16,7 @@ def ecliptic_coordinates
       Coordinates::Ecliptic.new(
         latitude: Angle.as_degrees(0),
         longitude: Angle.as_degrees(
-          (true_anomaly.value + longitude_at_perigee.to_degrees.value) % 360
+          (true_anomaly.degrees + longitude_at_perigee.degrees) % 360
         )
       )
```

Comment:
Out of scope for this first PR but a future refactor you could do would be to implement some math operators on `Angle`. Then you don't need to do the unwrap/do math/rewrap dance here. For example, if you implemented `Angle#+` and `Angle#%` this would become

```ruby
longitude: (true_anomaly + logitude_at_perigee) % 360
```

Reply:
The same goes for a lot of the math in this file. By implementing a few arithmetic operators on `Angle`, you get to work with angle values almost as if they were integers

Reply:
You will need to pay attention to the types when implementing math operations though. Keep in mind:

- Some operations return angles, some return raw integers
- Some operations work on two angles, some operations work on an angle and a number

For example `Angle#cos` would take an angle and return a float. `Angle#*` is effectively _scaling_ your angle. It takes an angle and a number and returns a new angle.

Reply:
I'd love to see a future world where this file doesn't do any unwrapping or re-wrapping of angles